src/Controller/Admin/PageExportController.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\ImportExportBundle\Controller\Admin;
+
+use Exception;
+use Neusta\Pimcore\ImportExportBundle\Documents\Export\PageExporter;
+use Neusta\Pimcore\ImportExportBundle\Toolbox\Repository\PageRepository;
+use Pimcore\Controller\UserAwareController;
+use Pimcore\Model\Document\Page;
+use Symfony\Component\HttpFoundation\HeaderUtils;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+final class PageExportController extends UserAwareController
+{
+    public function __construct(
+        private PageExporter $pageExporter,
+        private PageRepository $pageRepository,
+    ) {
+    }
+
+    /**
+     * @Route("/admin/page/export", name="page_export", methods={"GET"})
```

Comment:
We should prefix this with the bundle prefix:

```suggestion
     * @Route("/admin/neusta/import-export/page/export", name="neusta_pimcore_import_export_page_export", methods={"GET"})
```